### PR TITLE
silence

### PR DESCRIPTION
**Pull request by grosser**
@pschambacher @bquorning

### CODE REVIEW

File: lib/active_record_shards/model.rb [hunk 1]
```diff
@@ -24,13 +24,22 @@ def is_sharded? # rubocop:disable Style/PredicateName
     end
 
     def on_slave_by_default?
-      if base_class.instance_variable_defined?(:@on_slave_by_default)
-        base_class.instance_variable_get(:@on_slave_by_default)
+      if self == ActiveRecord::Base
+        false
+      else
+        base = base_class
```
Comment (bquorning) [inline lib/active_record_shards/model.rb]: I’m not sure it’s worth it to set a local variable.

Comment (grosser) [reply lib/active_record_shards/model.rb]: idk how expensive base_class is ... I'd wager reusing a local variable is faster then a noop method call ... and `base_class` might do all kinds of checks ... so opting out from that :)

Comment (bquorning) [reply lib/active_record_shards/model.rb]: Ahh, it’s a method call. I thought it was an attribute. Never mind then :-)

**Comment by kassio** [comment]
It seems that this one broke the build: https://travis-ci.com/zendesk/zendesk/jobs/89354125#L1057

**Comment by kassio** [comment]
Actually, my bad, this is not the reason. But with the last version of this, the broken test is green. 😊

**Pull request merged**